module/Kanbanize/Module.php

```diff
@@ -65,15 +65,16 @@ public function getServiceConfig()
 					return new StreamCommandsListener($entityManager);
 				},
 				'Kanbanize\ImportDirector' => function ($locator) {
-					$config = $locator->get('Config');
-					$apiKey	= $config['kanbanize']['apikey'];
 					$taskService = $locator->get('TaskManagement\TaskService');
 					$streamService = $locator->get('TaskManagement\StreamService');
 					$userService = $locator->get('Application\UserService');
 					$kanbanizeService = $locator->get('Kanbanize\KanbanizeService');
 					$transactionManager = $locator->get('prooph.event_store');
 					$service = new ImportDirector($kanbanizeService, $taskService, $streamService, $transactionManager, $userService);
-					$service->setApiKey($apiKey);
+					$config = $locator->get('Config');
+					if(is_string($config['kanbanize']['apikey'])){
```

Comment:
Questa condizione è sbagliata: nel caso non esista nel file di configurazione la chiave `kanbanize` genera un errore e la stessa cosa la fa in caso non esista `apikey`